packages/base-map/i18n/i18n-exceptions.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
It's not clear why this file has to be here. I think I know but it might be helpful to add to the readme or a comment somewhere

Reply:
Gorgeous. Thanks so much